src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncIteratorTests.cs

```diff
@@ -4780,10 +4780,219 @@ public static async System.Collections.Generic.IAsyncEnumerable<int> M()
 }";
             var comp = CreateCompilationWithAsyncIterator(new[] { Run(iterations), source }, options: TestOptions.DebugExe);
             comp.VerifyDiagnostics();
-            CompileAndVerify(comp, expectedOutput: expectedOutput);
+            var verifier = CompileAndVerify(comp, expectedOutput: expectedOutput,
+                verify: Verification.FailsILVerify with
+                {
+                    ILVerifyMessage = "[MoveNext]: Leave into try block. { Offset = 0x11a }"
```

Comment:
Is this failure message fine? I haven't seen it elsewhere.

Reply:
This is failing this way in main. It is unrelated to this PR.

Reply:
Link to tracking issue?

Reply:
https://github.com/dotnet/roslyn/issues/79124